etc/tempesta_fw.conf

```diff
@@ -1,264 +1,3 @@
-#   cache_db "/var/weird !;% name/foo.tdb";
-#
-# Default:
-#   cache_db /opt/tempesta/db/cache.tdb;
-
-# TAG: cache_size
-#
-# Size of file(s) created by Tempesta FW within cache_dir.
-#
-# Syntax:
-#   cache_size SIZE
-#
-# SIZE is specified in bytes, suffixes like 'MB' are not supported yet.
-# Also, the number must be a multiple of 2MB (Tempesta DB extent size).
-#
-# Default:
-#   cache_size 268435456;  # 256MB
-
-# TAG: filter_db
-#
-# Path to a filter database file used as a storage for Tempesta FW filter rules.
-# The same as cache_db.
-#
-# Default:
-#   filter_db /opt/tempesta/db/filter.tdb;
-
-# TAG: drop_tbl_size
-#
-# Size of filter drop table.
-#
-# Syntax:
-#   filter_tbl_size SIZE
-#
-# Default:
-#   filter_tbl_size 16777216;  # 16MB
-
-# TAG: sticky
-#
-# Tempesta sticky cookie.
-#
-# Syntax:
-#   sticky [name=<COOKIE_NAME>] [enforce]
-#
-# Default:
-#   Tempesta sticky cookie is not used.
-#
-# COOKIE_NAME is the name of Tempesta sticky cookie is used in HTTP
-# requests that pass through Tempesta. When not specified explicitly,
-# a default name is used.
-#
-# enforce - enforce the use of Tempesta sticky cookie in all HTTP
-# requests that come to Tempesta. If Tempesta sticky cookie is not
-# found in an HTTP request, a client get an HTTP 302 response that
-# redirects the client to the same URI, and prompts that Tempesta
-# sticky cookie is set in the request.
-#
-# Examples:
-#   sticky;		# Enable sticky cookie with default name.
-#   sticky enforce;	# Enforce sticky cookie with default name.
-#   sticky name=__cookie__ enforce;
-
-#
-# Frang configuration.
-#
-# TAG: frang_limits
-#
-# The section containing static limits for the classifier.
-#
-# Syntax:
-#   frang_limits {
-#	request_rate NUM;
-#	request_burst NUM;
-#	connection_rate NUM;
-#	connection_burst NUM;
-#	concurrent_connections NUM;
-#	client_header_timeout NUM;
-#	client_body_timeout NUM;
-#	http_uri_len NUM;
-#	http_field_len NUM;
-#	http_body_len NUM;
-#	http_host_required true|false;
-#	http_methods [METHOD]...;
-#	http_ct_required true|false;
-#	http_ct_vals ["CONTENT_TYPE"]...;
-#  }
-#
-#  - options with names *_rate define requests/connections rate per second.
-#  - *_burst are temporal burst for 1/FRANG_FREQ of second.
-#  - http_* are static limits for contents of a HTTP request.
-#
-# Example:
-#    frang_limits {
-#        request_rate 20;
-#        request_burst 15;
-#        connection_rate 8;
-#        connection_burst 6;
-#        concurrent_connections 8;
-#        client_header_timeout 20;
-#        client_body_timeout 10;
-#        http_uri_len 1024;
-#        http_field_len 256;
-#        http_ct_required false;
-#        http_methods get post head;
-#        http_ct_vals "text/plain" "text/html";
-#        http_header_chunk_cnt 10;
-#        http_body_chunk_cnt 0;
-#   }
-#
-# Default:
-#   All limits are disabled (the values are set to zero/false/empty).
-
-
+cache 0;
+listen 8081;
+server 127.0.0.1:80;
```

Comment:
The change breaks default configuration template. If you need exactly this config for the tests, then please place it in tempesta_fw/t/ directory.